### DISCUSSION TITLE
'Azure Pipelines' fileMatch to include yml files in .azuredevops directories

### DISCUSSION THREAD

**Discussion by SchulteDev** [renovatebot/renovate#31730] [Suggest an Idea]
### Tell us more.

Renovate supports updating Azure Pipelines https://docs.renovatebot.com/modules/manager/azure-pipelines
 
The `fileMatch` for time being is defined as: `azure.*pipelines?.*\\.ya?ml$`

**Suggestion**

In my experience, Azure Devops pipelines definitions are often stored in a folder `.azuredevops`

```bash
$ tree .azuredevops/
.azuredevops/
├── renovate.yml
├── run_tests.yml
```

Please add any yml/yaml file in an `.azuredevops` directory to the Azure Pipelines `fileMatch`.

**Comment by rarkins** [comment]
You're welcome to submit a PR with this change

**Reply by SchulteDev** [reply]
-> https://github.com/renovatebot/renovate/pull/31735
